index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Swipe</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="canvas">
+        <img  src="/image/Rectangle 1.png" alt="fondo" />
+    </div>
+    <div>
+        <nav class="travesia">
+            <img src="/image/logo-white.svg" alt="logo de la empresa">
+         
+            <ul class="menu">
+                <li><a href="#">Products</a></li>
+                <li><a href="#">Solutions</a></li>
+                <li><a href="#">Developers</a></li>
+                <li><a href="#">Resources</a></li>
+                <li><a href="#">Pricing</a></li>
+            </ul>
+                <div class="Hamburger">
+                <div class="line"></div>
+                <div class="line-two"></div>
+                <div class="line-three"></div>
+            </div>
+            <button class="boton-principal">Sing In</button>
+        </nav>
+    <main>
+        <div class="fortachon" >
+            <div class="container-twocolumns">
+                <div class="column-left">
+                    <h1> Payments infrastructure for the world</h1>
+                    <p>Tons of companies all over the world use Swipe’s  seamless platform to manage their businesses's entire financial transaction infrastructure online.</p>
+                    <nav class="buttons">
+                        <button class="first-button arrow">Start now<img class="flecha" src="/image/arrow-right-light.svg" alt="flecha"></button>
+                        <button class="second-button arrow">Contact Sales<img class="flecha"src="/image/arrow-right-purple.svg" alt="flecha"></button>
+                    </nav>
+
+                </div>
+                <div class="right" ><img class="phone" src="/image/hero-phone.png" alt="telefono"></div>
+            </div>
+            <div class="patrocinadores">
+                <div class="conteiner-patrocinadores">
+                    <div class="patrocinadores-grid">
+                        <img src="/image/instacart.png" alt="instacart">
+                        <img src="/image/salesforce.png" alt="salesforce">
+                        <img src="/image/amazon (1).png" alt="amazon">
+                        <img src="/image/slack.png" alt="slack">
+                        <img src="/image/shopify.png" alt="shopify">
+                        <img src="/image/google.png" alt="google">
+                        <img src="/image/lyft.png" alt="lyft">
+                        <img src="/image/zoom.png" alt="zoom">  
```

Comment:
Diego, en todas las partes en las que pusiste `/image` por favor borra el "/" al comienzo

```suggestion
                        <img src="image/instacart.png" alt="instacart">
                        <img src="image/salesforce.png" alt="salesforce">
                        <img src="image/amazon (1).png" alt="amazon">
                        <img src="image/slack.png" alt="slack">
                        <img src="image/shopify.png" alt="shopify">
                        <img src="image/google.png" alt="google">
                        <img src="image/lyft.png" alt="lyft">
                        <img src="image/zoom.png" alt="zoom">  
```